cloudapi/insights/proto/v1/common/common.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package k6.cloud.insights.proto.v1.common;
+
+option go_package = "go.k6.io/k6/cloudapi/insights/proto/v1/common";
+
+// Messages below are copied from https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/common/v1/common.proto
```

Comment:
Is this copy legit? Are we respecting the license?

Reply:
Not sure. I'm not a lawyer. Do we have somebody with more expertise in licensing that could chip in here?

Reply:
I added a copyright notice in [27137a9](https://github.com/grafana/k6/pull/3100/commits/27137a9a909b9b670d2be0c62554ee8c3d3cc73a). AFAIK, this should be enough, but perhaps someone with more knowledge could double-check it.